.github/workflows/ping.yaml

```
@@ -0,0 +1,15 @@
+name: Keep Render Alive (${{ secrets.PING_ENABLED == 'true' && 'ENABLED' || 'DISABLED' }})
+
+on:
+  schedule:
+    - cron: '*/14 23-15 * * *'  # KST 08:00 ~ 24:00
+  workflow_dispatch:
+
+jobs:
+  ping:
+    if: ${{ secrets.PING_ENABLED == 'true' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)